js/stores/appStore.js

```diff
@@ -239,7 +239,7 @@ function windowDefaults () {
     windowOffset: 20,
     webPreferences: {
       sharedWorker: true,
-      partition: ''
+      partition: 'main-1'
```

Comment:
are the autocomplete queries cacheable? That's really the only issue here because otherwise the in-memory main-1 partition is fine. I wonder if private partitions can use the cache from their corresponding persistent partition? Rather than being a data leak, that would actually improve privacy because fewer remote requests will be made.